src/test/java/uk/gov/hmcts/sptribs/common/notification/NotificationHelperTest.java

```diff
@@ -57,6 +59,9 @@ public class NotificationHelperTest {
     @InjectMocks
     private NotificationHelper notificationHelper;
 
+    @Mock
+    private CicCase cicCase;
+
     @Test
     void setRecordingTemplateVarsTest() {
```

Comment:
I would suggest the following test cases for the method setRecordingTemplateVars. (you can modify/remove the existing ones accordingly)
1. A test case which passes all the if conditions in that method
2. A test case which fails in all the if conditions in that method (selected venue not null)
3. A test case for CIC_CASE_HEARING_VENUE with HearingVenueNameAndAddress not null (not video or telephone format and selected venue null)
4. A test case for CIC_CASE_HEARING_VENUE with HearingVenueNameAndAddress null, not video or telephone format and selected venue null
5. A test case for CIC_CASE_HEARING_VENUE with video format, but not telephone format
6. A test case for CIC_CASE_HEARING_VENUE with telephone format, but not video format